terraso_backend/apps/graphql/schema/sites.py

```diff
@@ -301,6 +318,9 @@ def mutate_and_get_payload(cls, root, info, **kwargs):
                 old_projects.append(site.project)
 
         Site.bulk_change_project(to_change, project)
+        # update site depth intervals where necessary
```

Comment:
issue: Please explain more about what "when necessary" means. Under what circumstances does this occur?

Reply:
removed